src/pages/Pool/index.test.tsx

```diff
@@ -1,25 +1,25 @@
-import * as chains from 'constants/chains'
-import * as useV3Positions from 'hooks/useV3Positions'
+import { isSupportedChain } from 'constants/chains'
+import { useV3Positions } from 'hooks/useV3Positions'
+import { mocked } from 'test-utils/mocked'
 import { render, screen } from 'test-utils/render'
 
 import Pool from '.'
 
+jest.mock('constants/chains')
+jest.mock('hooks/useV3Positions')
```

Comment:
is this due to swc related mock restrictions?

Reply:
yes, but it's actually a more proper esm implementation - you shouldn't be able to mock a wildcard/glob import's properties. Instead, you should _have_ to use named imports if you are mocking.